webapp/channels/src/components/admin_console/ldap_wizard/ldap_wizard.scss

```diff
@@ -0,0 +1,93 @@
+.ldap-wizard-wrapper {
+    background-color: rgba(var(--center-channel-color-rgb), 0.04);
+
+    .ldap-wizard-content-wrapper {
+        display: flex;
+        overflow: hidden; // Prevent double scrollbars on the body if content overflows
+        flex-grow: 1;
+        padding-top: 20px;
+    }
+
+    
+    .ldap-wizard-sidebar {
+        position: sticky;
+        top: 0;
```

Comment:
I don't think we need the `position: sticky` or `top: 0` because this component isn't the one that scrolls

Reply:
Ah, good catch, thanks. Done.